src/accountsdb/snapshots.zig

```diff
@@ -923,6 +1078,171 @@ pub const BankFields = struct {
     }
 };
 
+/// Analogous to https://github.com/anza-xyz/agave/blob/8d1ef48c785a5d9ee5c0df71dc520ee1a49d8168/runtime/src/serde_snapshot.rs#L396
```

Comment:
would be good to include the rust struct name which its analogous too (ie, ExtraFieldsToDeserialize) -- this mainly helps with searching across clients (if you know the name in agave but want to see the equivalent in sig, you just search the rust struct)

Reply:
Oversight, good catch

Reply:
da113b56002e2d6bba09c235fc44a19ea9f2b848